app/models/gallery.rb

```diff
@@ -13,10 +15,15 @@ class Gallery < ActiveRecord::Base
 
   validates :title, presence: true, length: { maximum: 60 }
   validates :description, length: { maximum: 280 }
+  validates :slug, presence: true
```

Comment:
there's no field in the cms to enter this, can we add a before_save/create to set the slug to an URL friendly version of the title?